include/constants.hpp

```diff
@@ -89,12 +91,13 @@ constexpr auto kwdVpdInf = "com.ibm.ipzvpd.VINI";
 constexpr auto vsysInf = "com.ibm.ipzvpd.VSYS";
 constexpr auto kwdCCIN = "CC";
 constexpr auto kwdRG = "RG";
+constexpr auto kwdD0 = "D0";
 constexpr auto locationCodeInf = "com.ibm.ipzvpd.Location";
 constexpr auto pldmServiceName = "xyz.openbmc_project.PLDM";
 constexpr auto pimServiceName = "xyz.openbmc_project.Inventory.Manager";
 constexpr auto biosConfigMgrObjPath =
     "/xyz/openbmc_project/bios_config/manager";
-constexpr auto biosConfigMgrService = "xyz.openbmc_project.BIOSConfig.Manager";
+constexpr auto biosConfigMgrService = "xyz.openbmc_project.BIOSConfigManager";
```

Comment:
this service name is not used elsewhere in the code. can we add it in constants.hpp when required?

Reply:
It is being used. In D-Bus call. That call was tested with direct service name and later I replaced them with constants.

Reply:
ok